.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on: [push]
+
+jobs:
+  unittest:
+    name: Unit tests - ${{ matrix.PYTHON_VERSION }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION: ["3.7", "3.8"]
```

Comment:
out of curiosity, did you try 3.9 already?

Reply:
No, I would like to do that in a separate PR. My main intention with this PR is to get this going so one can work in parallel on the various issues.